pyvista/_vtk.py

```diff
@@ -520,6 +529,14 @@ def __init__(self):  # pragma: no cover
             from pyvista.core.errors import VTKVersionError
             raise VTKVersionError('vtkGLTFReader requires VTK v9 or newer')
 
+    class vtkPythonItem():  # type: ignore
+        """Empty placeholder for VTK9 compatibility."""
+
+        def __init__(self):  # pragma: no cover
+            """Raise version error on init."""
+            from pyvista.core.errors import VTKVersionError
+            raise VTKVersionError('Charts requires VTK v9 or newer')
+
```

Comment:
@akaszynski  Could you explain why this class is need?

Reply:
We need this because `charts.py` subclasses `vtkPythonItem` at runtime when creating Charts. If running VTK 8.1.2 and you attempt to create a chart, you'll get a nonsensical `ImportError`; better to get a `VTKVersionError`.

Reply:
I understand thanks.